NEWS

```diff
@@ -1,15 +1,83 @@
 #
-## Copyright (C) Mellanox Technologies Ltd. 2001-2015.  ALL RIGHTS RESERVED.
-## Copyright (C) UT-Battelle, LLC. 2014-2015. ALL RIGHTS RESERVED.
-## Copyright (C) ARM Ltd. 2017-2018.  ALL RIGHTS RESERVED.
+## Copyright (C) Mellanox Technologies Ltd. 2001-2020.  ALL RIGHTS RESERVED.
+## Copyright (C) UT-Battelle, LLC. 2014-2019. ALL RIGHTS RESERVED.
+## Copyright (C) ARM Ltd. 2017-2020.  ALL RIGHTS RESERVED.
 ##
 ## See file LICENSE for terms.
 ##
 #
 
 ## Current
+## 1.8.0-rc1 (TBD)
 Features:
-- TBD
+- Improved detection for DEVX support
+- Improved TCP scalability 
+- Added ROCM perftest
+- Added optimized memcpy for ROCM devices
+- Protection from TCP connection from non-UCX applications
+- Added hardware tag-matching for CUDA buffers
+- Added support for CUDA and ROCM managed memories
```

Comment:
to add:
- Support client/server disconnect protocol over rdma_cm
- Add support for striding receive queue for hardware tag-matching
- Xpmem-based rendezvous protocol for shared memory
- Enable using shared memory between containers on same machine
- Java API enhancements: @petro-rudenko pls add details
- Use multi-threaded RDMA memory registration for large regions

Reply:
Java API enhancements: 

- ucp_mmap by parameters functionality (ability to allocate + register, register buffer by address, incl. GPU buffers).
- Connection request functionality (create endpoint from conn handler)
- Recv size functionality.
- Support streaming API.
- Tag probe functionality.
- Publishing SNAPSHOT and release artifacts to maven central.
- Other bug fixes + improvements.

Reply:
@yosefe Shell we move Java to a separate section ?

Reply:
@yosefe can we qualify JUCX as beta quality API ? preview ?

Reply:
to add for Features section:
- Add Java API for stream send/recv, tag probe, and connect request handle
- Automatic publish of Java package to Maven central

> can we qualify JUCX as beta quality API ? preview ?
As preview (since API can change). Not sure how we can express this in NEWS

Reply:
@yosefe How we qualify the API ? Beta ? preview ?

Reply:
Preview